examples/experiments/decay/research_decay.ipynb

```diff
@@ -0,0 +1,382 @@
+{
```

Comment:
- Add the same decay as in `self-attention` notebook
- That is not how you supposed to use `cyclic lr`, I suppose 
- The deviation in your results is an order of magnitude higher, and that is very suspicious
- Fix formatting of the third cell
- Spend more time thinking which decays you want to showcase: most of the current are really weird